test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -19,8 +19,9 @@ function usage() {
     Push the latest Antrea image to all vagrant nodes and restart the Antrea daemons
           --prometheus                 Deploy Prometheus service to scrape metrics
                                        from Antrea Agents and Controllers.
-          --flow-collector <Addr|ELK>  Provide either the external IPFIX collector
-                                       address or specify 'ELK' to deploy the ELK
+          --flow-collector <Addr|ELK|clickhouse>
+                                       Provide either the external IPFIX collector
+                                       address or specify 'ELK' or 'clickhouse' to deploy the ELK
```

Comment:
or specify 'ELK' to deploy the ELK flow collector, or specify 'Grafana' to deploy the Grafana flow collector

Reply:
I've updated network-flow-visibility.md.

Reply:
Cool, could you also help fix one typo in the doc on https://github.com/antrea-io/antrea/blob/7c6042bb5440cb5ed356495d3e1e5c010e638cae/docs/network-flow-visibility.md#L593, `xxx/clickhouse-operator-install-bundle.yaml` -> `xxx/clickhouse-operator-install-bundle.yml`. Thanks!